packages/core/src/shipping/shipping-address-selector.ts

```diff
@@ -36,10 +38,35 @@ export function createShippingAddressSelectorFactory(): ShippingAddressSelectorF
         },
     );
 
+    const getShippingAddresses = createSelector(
```

Comment:
Hey @Tharaae sorry just noticed this, can we add tests to cover this as well please. Thanks.